packages/edit-navigation/src/components/inspector-additions/delete-menu-panel.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default function DeleteMenuPanel( { onDeleteMenu } ) {
+	return (
+		<PanelBody className="edit-navigation-inspector-additions__delete-menu-panel">
+			<Button
+				isLink
+				isDestructive
+				onClick={ () => {
+					if (
+						// eslint-disable-next-line no-alert
+						window.confirm(
```

Comment:
Should we be using `window.confirm`? If this is temporary code only, maybe we should add a `FIXME` comment above?

Reply:
We don't yet have a better alternative. See https://github.com/WordPress/gutenberg/issues/16583. I don't think a `FIXME` is very necessary because it will come up when the hypothetical future fixer of https://github.com/WordPress/gutenberg/issues/16583 searches for `window.confirm`.

Reply:
Oh I didn't know #16583 exists, thanks!